### PR TITLE
<AdaptiveHeading> - add component

### PR DESCRIPTION
**Pull request by maxvektor**

### CODE REVIEW

File: .wuf/components.json [hunk 1]
```diff
@@ -388,5 +388,8 @@
   },
   "LabelledElement": {
     "path": "src/LabelledElement"
+  },
+  "AdaptiveHeading": {
```
Comment (argshook) [inline .wuf/components.json]: remove

File: .wuf/testkits/definitions.js [hunk 2]
```diff
@@ -774,5 +774,10 @@ module.exports = {
   FillPreview: {
     unidriver: true,
     testkitPath: "../src/FillPreview/FillPreview.uni.driver"
+  },
+
+  AdaptiveHeading: {
```
Comment (argshook) [inline .wuf/testkits/definitions.js]: remove

File: src/AdaptiveHeading/AdaptiveHeading.meta.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import AdaptiveHeading from './AdaptiveHeading';
```
Comment (argshook) [inline src/AdaptiveHeading/AdaptiveHeading.meta.js]: remove

File: src/AdaptiveHeading/test/AdaptiveHeading.private.uni.driver.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { adaptiveHeadingDriverFactory as publicDriverFactory } from '../AdaptiveHeading.uni.driver';
```
Comment (argshook) [inline src/AdaptiveHeading/test/AdaptiveHeading.private.uni.driver.js]: remove

File: src/AdaptiveHeading/test/AdaptiveHeading.visual.js [hunk 5]
```diff
@@ -0,0 +1,116 @@
+import React from 'react';
```
Comment (argshook) [inline src/AdaptiveHeading/test/AdaptiveHeading.visual.js]: remove

File: src/AdaptiveHeading/test/storySettings.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+import { Category } from '../../../stories/storiesHierarchy';
```
Comment (argshook) [inline src/AdaptiveHeading/test/storySettings.js]: remove

File: stories/index.js [hunk 7]
```diff
@@ -197,3 +197,6 @@ require('../src/Deprecated/SideMenu/docs/SideMenu');
 
 // BETA
 require('../src/beta/PopoverMenu/docs/index.story');
+
+// TODO: move to correct position
+require('../src/AdaptiveHeading/docs/index.story');
```
Comment (argshook) [inline stories/index.js]: remove

File: testkit/component-definitions.js [hunk 8]
```diff
@@ -280,4 +280,10 @@ export default {
       onClose: () => {},
     },
   },
+
+  AdaptiveHeading: {
```
Comment (argshook) [inline testkit/component-definitions.js]: remove

File: testkit/enzyme.d.ts [hunk 9]
```diff
@@ -127,6 +127,7 @@ declare namespace EnzymeTestkit {
   export const statisticsWidgetTestkitFactory: any;
   export const fillPreviewTestkitFactory: any;
   export const labelledElementTestkitFactory: any;
+  export const adaptiveHeadingTestkitFactory: any;
```
Comment (argshook) [inline testkit/enzyme.d.ts]: run `npm run update-components` to remove the rest of the changes.

they are automated and will be gone, once `AdaptiveHeading` is moved to `src/utils`

File: src/AdaptiveHeading/docs/index.story.js [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+            ),
+          ]),
+
+          divider(),
+
+          title('Examples'),
+
+          columns([
+            description({
+              title: 'Usage with long numbers',
```
[the remark this answers is not in the document]
Comment (argshook) [reply src/AdaptiveHeading/docs/index.story.js]: ignore, this file should be removed

File: testkit/protractor.js [hunk 11]
```diff
@@ -579,6 +579,8 @@ import fillPreviewDriverFactory from '../src/FillPreview/FillPreview.uni.driver'
 
 export const fillPreviewTestkitFactory = protractorUniTestkitFactoryCreator(
   fillPreviewDriverFactory,
+);
```
Comment (maxvektor) [inline testkit/protractor.js]: looks like this and bellow doesn't belong to this PR.
